Client/Assets/ContextMenu.js

```diff
@@ -74,10 +74,37 @@ addEventListener("touchstart", function (event) {
   var data = {};
   var element = event.target;
 
+  element.addEventListener("touchend", function (event) {
```

Comment:
We should also make sure to remove these event listeners so that they don't build up every time an element is touched. Easiest way to do that would be to 1) make this a separate named function, then 2) call `element.removeEventListener(...)`with it. We should probably remove both listeners in `touchend` (since there's no guarantee `touchmove` will be called).